### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kchygoe**
## What this patch does to fix the issue.
Update dependencies including security updates
- Update blueoil package deps
- Update output_template requirements
- Change de10-nano test env to adopt backward compatibility for other PRs
- (test env should be changed back again after CI env packages are all updated)

## Link to any relevant issues or pull requests.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -19,37 +19,37 @@ packages = find:
...
+  horovod==0.19.5
+  mpi4py==3.0.3
 docs =
   better-apidoc==0.3.1
   sphinx-rtd-theme==0.2.4
   Sphinx==2.4.4
   sphinx-autobuild==0.7.1
 test =
-  flake8==3.7.9
+  flake8==3.3.3
```
Comment (iizukak) [inline setup.cfg:48]: Downgrade ? 🤔

Comment (kchygoe) [reply setup.cfg:48]: !!

Comment (kchygoe) [reply setup.cfg]: 😂

**Pull request merged**